packages/common/src/types/savedCharts.ts

```diff
@@ -3,6 +3,16 @@ import { CompactOrAlias } from './field';
 import { MetricQuery } from './metricQuery';
 import { UpdatedByUser } from './user';
 
+export enum ChartKind {
```

Comment:
does anyone have a better name for this enum ? (ChartType is taken already)

Reply:
I always use Kind when Type is not available 😂